blockchain/vm/memory.go

```diff
@@ -133,6 +133,9 @@ func (m *Memory) Print() {
 }
 
 func (m *Memory) Slice(from, to int64) []byte {
+	if from == to {
```

Comment:
```suggestion
	if from >= to {
```

Reply:
Since the value can be negative, let's leave it as is for now. We need to revisit with the other code:
https://github.com/klaytn/klaytn/blob/dev/node/cn/tracers/tracer.go#L100-L116